Update the Ethereum client guide (#987)
smartcontractkit/documentation: docs/chainlink-nodes/run-an-ethereum-client.md

layout: nodes.liquid
section: nodeOperator
date: Last Modified
title: 'Run an Ethereum Client'
permalink: 'docs/run-an-ethereum-client/'
whatsnext:
  {
    'Running a Chainlink Node': '/docs/running-a-chainlink-node/',
    'Optimizing Performance': '/docs/evm-performance-configuration/',
  }
---

> Chainlink nodes must be able to connect to an Ethereum client with an active websocket connection. This is accomplished by running both an execution client and a concensus client. You can run these clients yourself, butrunning Ethereum clients requires significant storage and network resources. Optionally, you can use [External Services](#external-services) that handle management of these clients for you.

Reviewer: ```suggestion
Chainlink nodes must be able to connect to an Ethereum client with an active websocket connection. This is accomplished by running both an execution client and a consensus client. You can run these clients yourself, but running Ethereum clients requires significant storage and network resources. Optionally, you can use [External Services](#external-services) that manage these clients for you.
```
